### ISSUE TITLE
TS Server fatal error:  Cannot read property 'indexOf' of undefined

### ISSUE THREAD

**Issue by nitpum** [microsoft/vscode#116426]
Issue Type: <b>Bug</b>

❗️❗️❗️ Please fill in the sections below to help us diagnose the issue ❗️❗️❗️

**TypeScript Version:** 4.1.4

**Steps to reproduce crash**
1. Open VSCode
2. Open typescript file on WSL2

**TS Server Log**
❗️ Please review and upload this log file to help us diagnose this crash:
`/home/nitpum/.vscode-server/data/logs/20210211T153455/exthost3/vscode.typescript-language-features/tsserver-log-MDCurO/tsserver.log`
The log file may contain personal data, including full paths and source code from your workspace. You can scrub the log file to remove paths or other personal information.

**TS Server Error Stack**
Server: `semantic`
```
TypeError: Cannot read property 'indexOf' of undefined
    at Object.parsePackageName (/home/nitpum/.vscode-server/bin/5d424b828ada08e1eb9f95d6cb41120234ef57c7/extensions/node_modules/typescript/lib/tsserver.js:39472:30)
    at ConfiguredProject.Project.enablePlugin (/home/nitpum/.vscode-server/bin/5d424b828ada08e1eb9f95d6cb41120234ef57c7/extensions/node_modules/typescript/lib/tsserver.js:151962:24)
    at ConfiguredProject.enablePluginsWithOptions (/home/nitpum/.vscode-server/bin/5d424b828ada08e1eb9f95d6cb41120234ef57c7/extensions/node_modules/typescript/lib/tsserver.js:152477:30)
    at ProjectService.loadConfiguredProject (/home/nitpum/.vscode-server/bin/5d424b828ada08e1eb9f95d6cb41120234ef57c7/extensions/node_modules/typescript/lib/tsserver.js:154206:25)
    at ProjectService.createAndLoadConfiguredProject (/home/nitpum/.vscode-server/bin/5d424b828ada08e1eb9f95d6cb41120234ef57c7/extensions/node_modules/typescript/lib/tsserver.js:154145:22)
    at ProjectService.createLoadAndUpdateConfiguredProject (/home/nitpum/.vscode-server/bin/5d424b828ada08e1eb9f95d6cb41120234ef57c7/extensions/node_modules/typescript/lib/tsserver.js:154150:36)
    at ProjectService.assignProjectToOpenedScriptInfo (/home/nitpum/.vscode-server/bin/5d424b828ada08e1eb9f95d6cb41120234ef57c7/extensions/node_modules/typescript/lib/tsserver.js:155055:44)
    at /home/nitpum/.vscode-server/bin/5d424b828ada08e1eb9f95d6cb41120234ef57c7/extensions/node_modules/typescript/lib/tsserver.js:155395:97
    at Object.flatMap (/home/nitpum/.vscode-server/bin/5d424b828ada08e1eb9f95d6cb41120234ef57c7/extensions/node_modules/typescript/lib/tsserver.js:513:25)
    at ProjectService.applyChangesInOpenFiles (/home/nitpum/.vscode-server/bin/5d424b828ada08e1eb9f95d6cb41120234ef57c7/extensions/node_modules/typescript/lib/tsserver.js:155395:41)
    at Session.handlers.ts.Map.ts.getEntries._a.<computed> (/home/nitpum/.vscode-server/bin/5d424b828ada08e1eb9f95d6cb41120234ef57c7/extensions/node_modules/typescript/lib/tsserver.js:156413:46)
    at /home/nitpum/.vscode-server/bin/5d424b828ada08e1eb9f95d6cb41120234ef57c7/extensions/node_modules/typescript/lib/tsserver.js:158286:88
    at IOSession.Session.executeWithRequestId (/home/nitpum/.vscode-server/bin/5d424b828ada08e1eb9f95d6cb41120234ef57c7/extensions/node_modules/typescript/lib/tsserver.js:158277:28)
    at IOSession.Session.executeCommand (/home/nitpum/.vscode-server/bin/5d424b828ada08e1eb9f95d6cb41120234ef57c7/extensions/node_modules/typescript/lib/tsserver.js:158286:33)
    at IOSession.Session.onMessage (/home/nitpum/.vscode-server/bin/5d424b828ada08e1eb9f95d6cb41120234ef57c7/extensions/node_modules/typescript/lib/tsserver.js:158310:35)
    at Interface.<anonymous> (/home/nitpum/.vscode-server/bin/5d424b828ada08e1eb9f95d6cb41120234ef57c7/extensions/node_modules/typescript/lib/tsserver.js:160520:27)
    at Interface.emit (events.js:315:20)
    at Interface._onLine (readline.js:329:10)
    at Interface._normalWrite (readline.js:474:12)
    at Socket.ondata (readline.js:186:10)
    at Socket.emit (events.js:315:20)
    at addChunk (_stream_readable.js:295:12)
    at readableAddChunk (_stream_readable.js:271:9)
    at Socket.Readable.push (_stream_readable.js:212:10)
    at Pipe.onStreamRead (internal/stream_base_commons.js:186:23)
```

VS Code version: Code 1.53.1 (5d424b828ada08e1eb9f95d6cb41120234ef57c7, 2021-02-08T23:29:42.785Z)
OS version: Windows_NT x64 10.0.19042
Remote OS version: Linux x64 4.19.104-microsoft-standard

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-8750H CPU @ 2.20GHz (12 x 2208)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>protected_video_decode: enabled<br>rasterization: enabled<br>skia_renderer: enabled_on<br>video_decode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|31.80GB (18.80GB free)|
|Process Argv|--crash-reporter-id 9fa0e17a-849f-41f9-962c-bbd2fdf8ed40|
|Screen Reader|no|
|VM|0%|

|Item|Value|
|---|---|
|Remote|WSL: Ubuntu-20.04|
|OS|Linux x64 4.19.104-microsoft-standard|
|CPUs|Intel(R) Core(TM) i7-8750H CPU @ 2.20GHz (12 x 2208)|
|Memory (System)|24.90GB (23.03GB free)|
|VM|0%|
</details><details><summary>Extensions (23)</summary>

Extension|Author (truncated)|Version
---|---|---
emojisense|bie|0.8.0
bracket-pair-colorizer-2|Coe|0.2.0
auto-rename-tag|for|0.1.6
vue-snippets|hol|1.0.4
vscode-peacock|joh|3.9.1
dotenv|mik|1.0.1
vscode-edge-devtools|ms-|1.1.3
remote-containers|ms-|0.158.0
remote-ssh|ms-|0.63.0
remote-ssh-edit|ms-|0.63.0
remote-wsl|ms-|0.53.4
vscode-remote-extensionpack|ms-|0.20.0
debugger-for-edge|msj|1.0.15
biblatex-snippets|nik|1.0.5
proto|pet|0.0.2
material-icon-theme|PKi|4.5.0
code-settings-sync|Sha|3.4.3
gitconfig|sid|2.0.1
swdc-vscode|sof|2.4.7
errorlens|use|3.2.4
highlight-matching-tag|vin|0.10.0
gistfs|vsl|0.2.2
mono-snippets|zra|0.0.2

(4 theme extensions excluded)

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
python383cf:30185419
vspyt653:30253241
vspor879:30202332
vspor708:30202333
vspor363:30204092
vswsl492cf:30211402
pythonvsdeb440:30248342
pythonvsded773:30248341
pythonvspyt600cf:30251589

```

</details>

**Comment by nitpum** [comment]
Same as #116425

**Comment by IllusionMH** [comment]
/duplicate of #116219 
Fix should go out with VS Code 1.53.2
